### PR TITLE
Add topic "coverage"

### PR DESCRIPTION
**Pull request by shaedrich**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

Coverage comes in many shapes and forms, however, it's nearly as important as testing itself across the board.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/coverage/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+display_name: Coverage
```
Comment (ipc103) [inline topics/coverage/index.md:2]: Just my opinion, but I think `Code coverage` should be the default topic. What do you think?

Comment (shaedrich) [reply topics/coverage/index.md:2]: I actually thought about that, too, but refrained from doing so as there is "code coverage" on one hand and "test coverage" on the other. What do you think about that?

Comment (ipc103) [reply topics/coverage/index.md:2]: `Test coverage` might be a little bit better, but I'm okay with either one. I just think `Coverage` on its own is too generic to be clear out of context.

Comment (ipc103) [reply topics/coverage/index.md:2]: I think this is most similar to the `react` vs `reactjs` example around aliases: https://github.com/github/explore/blob/3da6134c57074effddbc61e644502dacb6afc595/docs/API.md?plain=1#L5

For this one, I'd recommend using `Test coverage` as the topic name, with `coverage` and `code-coverage` as aliases. What do you think? I'm open to `Code coverage` as the topic name as well, but I think the aliases should cover us either way.

**Comment by Rohiyo** [comment]
در تاریخ چهارشنبه ۱۱ اکتبر ۲۰۲۳،‏ ۲۱:۴۳ Ian Candy ***@***.***>
نوشت:

File: topics/coverage/index.md (hunk 1 above)
Comment (shaedrich) [reply topics/coverage/index.md:2]: https://www.browserstack.com/guide/code-coverage-vs-test-coverage
https://www.lambdatest.com/blog/code-coverage-vs-test-coverage/

I don't think, this is comparable to your example. But after all, it's your decision 🤷🏻‍♂️

Comment (shaedrich) [reply topics/coverage/index.md:2]: As this is probably the closest we'll come to an agreement, I just say, I'll make the necessary changes.

**Review by ipc103 — APPROVED** [review]
Thanks for making those changes

**Comment by shaedrich** [comment]
You're welcome

**Pull request merged**